README.md

```diff
@@ -93,18 +93,26 @@ pip install polaris-lib
 
 ### Setup dev environment
 
-```bash
+```shell
 conda env create -n polaris -f env.yml
 conda activate polaris
 
 pip install --no-deps -e .
 ```
 
+Alternatively, using [uv](https://github.com/astral-sh/uv):
+```shell
+uv venv -p 3.12 polaris
+source .venv/polaris/bin/activate
+uv pip compile pyproject.toml -o requirements.txt --all-extras
+uv pip install -r requirements.txt 
+```
```

Comment:
By default, I would like to keep the README concise to not overwhelm with too much information. I would thus recommend a single way of installing it, but offer alternative installation options through a expandable component (similar to e.g. [Pytorch Lightning](https://github.com/Lightning-AI/pytorch-lightning) does it!). 

Something like: 

```suggestion
<details>
  <summary>Other installation options</summary>
  
    Alternatively, using [uv](https://github.com/astral-sh/uv):
    ```shell
    uv venv -p 3.12 polaris
    source .venv/polaris/bin/activate
    uv pip compile pyproject.toml -o requirements.txt --all-extras
    uv pip install -r requirements.txt 
    ```   
</details>

```